### PR TITLE
fixes #198, #199

### LINKED ISSUE #198: #1508
Create Tool tips for users

### PR DESCRIPTION
**Pull request by tcavaletto**
Tooltips done.

### CODE REVIEW

File: app/views/dashboard/dashboard_main.html.erb [hunk 1]
```diff
@@ -0,0 +1,72 @@
+<main class="container">
```
Comment (bammons) [inline app/views/dashboard/dashboard_main.html.erb]: This file shouldn't be here

File: app/views/curricula/history/show.html.erb [hunk 2]
```diff
@@ -7,14 +7,14 @@
         <li class="list-group-item">
           <div class="row">
             <div class="col-md-6">
-              <%= link_to curricula_compare_path(:id => @curriculum.id, :commit => c) do %>
+              <%= link_to curricula_compare_path(:id => @curriculum.id, :commit => c), title:"Click this to look at your save", rel:"tooltip" do %>
```
Comment (kdorer) [inline app/views/curricula/history/show.html.erb]: This should probably sound more like "Click this to compare this save with the previous one."

File: app/views/dashboard/dashboard_main.html.erb (hunk 1 above)
Comment (kdorer) [inline app/views/dashboard/dashboard_main.html.erb]: In my recent push I renamed 'dashboard_main' to 'show'. This file is probably left over from your merge.

**Comment by kdorer** [comment]
-1. Fix those problems, then push again and the test will rerun. The build failed on bundle which is not your problem.

**Comment by kdorer** [comment]
Another branch was just merged, you will have to pull down.

**Comment by kdorer** [comment]
+1

**Comment by Linkster700** [comment]
+1

**Pull request merged**